### ISSUE TITLE
Esri Amazon REST services are down for ERG

### ISSUE THREAD

**Issue by kevinsagis** [Esri/solutions-webappbuilder-widgets#926]
### Widget

ERG

### Version of widget

Downloaded as of 8/3/17 run on WAB 2.4

### Bug or Enhancement

REST services are down or missing or moved ??? 404

http://arcgis-emergencymanagement-2057568539.us-east-1.elb.amazonaws.com/arcgis/rest/services/COP/ERGByPlacard/GPServer/ERG%20By%20Placard

etc from the Config are all down or missing ....

**Comment by kgonzago** [comment]
Those AWS services are no longer available. There is a SD file containing those services in the [services](https://github.com/Esri/solutions-webappbuilder-widgets/tree/dev/ERG/services) directory.

**Comment by kevinsagis** [comment]
Hi Kevin,

What do I do with these?  SD is service definition. I understand. I have a server and we have an AWS server and an Org acct. Am I supposed to publish something? As a GP service maybe?  I have only ever published services from my own MXDs. I am not familiar with working with an SD file nor have really seen one. (I know only that the mxd becomes and msd when I publish)

However, the ERG would be extremely useful in disasters. I have my OSHA 40 HAZWOPER and have worked an EOC before, know of ALOHA etc and fully understand and will convey the limitations. But it could be a useful ballpark estimate. Or even just useful for training exercises. So how could we employ this ERG widget and its needed services? If you would like you could call any time 912-651-1452. I am happy to test anything you all would like, etc.

Thank you very much for your time and reply,
Best regards,
Kevin
912 651 1452
www.sagis.org<http://www.sagis.org>

**Comment by kgonzago** [comment]
In either ArcCatalog or ArcMap, you can right-click on a SD file and publish them to your ArcGIS Server.

**Comment by kevinsagis** [comment]
Hi Kevin,

Thanks. So I published it.

http://sagisservices.thempc.org/saint/rest/services/CEMA/EmergencyOperations/MapServer
So it was set up for Local Gov’t Data Model, and for Naperville, IL, I see.  Was it ever a turnkey solution?  I was thinking, I put a point on the map, and based on wind direction and hazmat material type, and maybe other met. Conditions from NWS, it would create an ALOHA plume. I was thinking maybe it could do that, were I to throw it into WAB.  And show demographics in it from the Census. I do not care about facilities and the other local data, really. Just for creating the plume overlay, for a visual.

Would we need to have the full Local Data model etc? I also don’t see where to put the FindNearestWS/GPServer<http://sagisservices.thempc.org/saint/rest/services/CEMA/FindNearestWS/GPServer>. Nothing resembles that REST endpoint in the Config of the ERG Widget. Though I presume it’d go in Weather Station GP Server the third one. It’s just that it does not match the Amazon URL in format, so I was wondering.

Thank you again for the help and your time! 😊

-Kevin

[cid:image003.jpg@01D311F7.0D3B9800]
[cid:image004.jpg@01D311F7.0D3B9800]

**Comment by kgonzago** [comment]
The SD files were meant to be a guide to creating your own facilities layer for example. And yes the associated data are centered on Naperville, IL. 

As for creating ALOHA plumes, that functionality does not exist in this widget. You will need to create that yourself and then brought in as a layer. [Robert Sheitlin](https://geonet.esri.com/thread/119278-roberts-custom-wab-widgets) has created something similar: [Aloha Threat Zone Widget](https://geonet.esri.com/docs/DOC-7733-aloha-threat-zone-widget-version-23-021417). Take a look at that widget along with the [try it live site](http://gis.calhouncounty.org/WAB/V2.5/widgets/Aloha/index.html).

**Comment by kevinsagis** [comment]
Indeed, I will probably integrate his widget. Ideally if ALOHA and/or this widget’s functionality (which is different as you denote) was integrated by EPA/NOAA perhaps partnering with Esri or others as an out-of-the-box widget that created plumes without needing the separate desktop ALOHA it would be a lot more accessible. As a complete turnkey web-based solution it would be useable by EOC duty officers, who have understanding of ERG and hazmat response. (i.e. HAZWOPER, firefighter, utilities ESF staff)  This is a comment more for EPA/NOAA but why I went looking and came across this widget. Thank you for the reply!  I will look in to Robert’s widget.

Kevin
